test/cmd.js

```diff
@@ -335,35 +381,6 @@ describe('express(1)', function () {
       .expect(404, /<h1>Not Found<\/h1>/, done);
     });
   });
-
-  describe('--help', function () {
```

Comment:
I really want to accept the pull request, but I can't as long as we keep messing with the help test. Now it tests both, but when one is broken, you can't tell if -h is broken or --help is broken based on the test output and line numbers...

Please just keep all the help tests as-is for now. Perhaps make another pull request with proposed changes here, so I can accept the other parts of this pull request that matches the title.